chsdi/locale/de/LC_MESSAGES/chsdi.po

```diff
@@ -1442,18 +1442,288 @@ msgstr "Zählstelle"
 msgid "ch.astra.unfaelle-personenschaeden_alle"
 msgstr "Unfälle mit Personenschaden"
 
+msgid "ch.astra.unfaelle-personenschaeden_alle.accidentday_de"
+msgstr "Unfalltag"
+
+msgid "ch.astra.unfaelle-personenschaeden_alle.accidentday_fr"
+msgstr "Unfalltag"
+
+msgid "ch.astra.unfaelle-personenschaeden_alle.accidentday_it"
+msgstr "Unfalltag"
```

Comment:
Some dependecy with the query tool here I think and a naming convention for the request to discover which attributes are queryable.
See: http://api3.geo.admin.ch/rest/services/ech/MapServer/ch.swisstopo.geologie-gravimetrischer_atlas.metadata

I believe this could be improved at the same we address your other comment on this PR.